app/vmagent/remotewrite/client_test.go

```diff
@@ -0,0 +1,72 @@
+package remotewrite
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/golang/snappy"
+	"github.com/stretchr/testify/assert"
```

Comment:
We don't use extra libraries for tests. Could you update the test so it won't use `testify/assert`>?

Reply:
Since empty block is skipped before send,  this test file is useless and has been deleted.